Tests/AGSnapshotHelperTests/FBSnapshotTestCase+AGSnapshotHelperTests.m

```diff
@@ -24,28 +24,29 @@ - (void)setUp {
 }
 
 - (void)testTestViewWithDisabledRecodedModeShouldCompareView {
-    XCTAssertNoThrow([self testView:[self exampleView] contentSizeCategory:UIContentSizeCategoryLarge]);
+    XCTAssertNoThrow([self testView:[self exampleView]]);
 }
 
-- (void)testTestLayerCategoryWithDisabledRecodedModeShouldCompareView {
-    XCTAssertNoThrow([self testLayer:[self exampleLayer] contentSizeCategory:UIContentSizeCategoryLarge]);
+- (void)testTestViewWithDisabledRecodedModeAndContentSizeCategoryShouldCompareView {
+    AGSampleView *view = [[AGSampleView alloc] initWithFrame:CGRectMake(0.0f, 0.0f, 320.0f, 568.0f)];
+    XCTAssertNoThrow([self testView:view contentSizeCategory:UIContentSizeCategoryExtraSmall]);
+}
+
+
+- (void)testTestViewWithDisabledRecodedModeAndContentSizeCategoryAndWindowBoundsSizesMaskShouldCompareView {
+    AGSampleView *view = [[AGSampleView alloc] initWithFrame:CGRectZero];
+    XCTAssertNoThrow([self testView:view contentSizeCategory:UIContentSizeCategoryExtraSmall windowsBoundsSizesMask:AGWindowBoundsSizesiPad]);
 }
 
 #pragma mark - Helper methods
 
 - (UIView *)exampleView {
+    CGRect AGViewFrame = (CGRect){{0.0f, 0.0f}, {50.0f, 50.0f}};
+
     UIView *exampleView = [[UIView alloc] initWithFrame:AGViewFrame];
     exampleView.backgroundColor = [UIColor redColor];
 
     return exampleView;
 }
 
-- (CALayer *)exampleLayer {
-    CALayer *exampleLayer = [[CALayer alloc] init];
-    exampleLayer.frame = AGViewFrame;
-    exampleLayer.backgroundColor = [UIColor blueColor].CGColor;
-
-    return exampleLayer;
-}
-
 @end
```

Comment:
Doesn't matter. If background would increase readability I can add. Would for you?